src/detectors/angular.js

```diff
@@ -0,0 +1,31 @@
+const { hasRequiredDeps, hasRequiredFiles, scanScripts } = require('./utils/jsdetect')
+
+module.exports = function() {
+  // REQUIRED FILES
+  if (!hasRequiredFiles(['package.json'])) return false
+  if (!hasRequiredFiles(['angular.json'])) return false
+  // REQUIRED DEPS
+  if (!hasRequiredDeps(['@angular/cli'])) return false
+
+  /** everything below now assumes that we are within angular */
+
+  const possibleArgsArrs = scanScripts({
+    preferredScriptsArr: ['serve', 'start', 'build'],
+    preferredCommand: 'ng build --prod'
+  })
+
+  if (possibleArgsArrs.length === 0) {
+    // offer to run it when the user does not have any scripts setup! 🤯
+    possibleArgsArrs.push(['ng', 'build', '--prod'])
+  }
+
+  return {
+    type: '@angular/cli',
+    port: 8888,
```

Comment:
The `command` property is actually required for everything to work correctly. Please revert your last commit. Otherwise everything looks great! 🎉